Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+@Library(value = "tomtom-nav-pipeline-shared-library@1.2.0") _
```

Comment:
Not sure I want to use Travis-CI for it. Running it on our Jenkins infra means that we will have everything in one place and can control and see better what kind of projects we have and build.

Even if we would use Jenkins, contributions from others will still be validated as our infra would be used. So as long as there is no real reason to use travis let's stick to our infra.

But if you have some concrete contra-arguments I am open to change it. If yes, then you have to create an account on travis-ci as I don't have any permissions on the github tomtom-international organization.

Reply:
I can think of couple reasons for using public CI instead of internal one.
* The CI builds and runs tests, so it produces some artifacts that won't be available if we'll be using internal CI. (e.g. test results, coverage, etc.)
* We shouldn't be running any steps in the pipeline which definition are not visible (and they wouldn't be if they are defined internally) - just for transparency
* Finally it's just simple to use public cloud that is always available and not blocking our internal resources

Reply:
Sounds plausible. Will introduce a travis config.

Reply:
Remember about squashing commits before merging. :-)

Reply:
Created travis configuration and squashed all commits